src/Razor/src/Microsoft.VisualStudio.RazorExtension/ServiceHubCore/Microsoft.VisualStudio.Razor.TagHelperProviderCore64.servicehub.service.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+  "host": "coreClr",
+  "hostId": "RoslynCodeAnalysisService",
+  "hostGroupAllowed": true,
+  "entryPoint": {
+    "assemblyPath": "Microsoft.CodeAnalysis.Remote.Razor.dll",
+    "fullClassName": "Microsoft.CodeAnalysis.Remote.Razor.RemoteTagHelperProviderServiceFactory",
+    "AssemblySearchPathsConfig": "Microsoft.VisualStudio.Razor.TagHelperProvider.AssemblySearchPathsConfig.json"
```

Comment:
Ideally we shouldn't need to set AssemblySearchPathsConfig to point to roslyn extension folder, since we already declared Roslyn as friendServices, thus resolving any dependencies from roslyn should be handled by Roslyn's ALC.
This is here only to workaround an issue that if load the same assembly from different locations in different ALC's that are trying to work together. @RyanToth3 is working on a solution